french/feu-de-camp.md

```diff
@@ -16,4 +16,5 @@ Que faites-vous ?
 
 [Je dessine une porte avec un syphon de cr�me chantilly qui traine](dessine/porte.md)
 
-[Je ne parle pas francais](../english/marshmallow.md)
\ No newline at end of file
+[Je ne parle pas francais](../english/marshmallow.md)
+[J'appelle ma m�re imm�diatement!](appel-maman/appeler-maman.md)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.